lab_1_classify_by_unigrams/main.py

```diff
@@ -49,6 +70,13 @@ def create_language_profile(language: str, text: str) -> dict[str, str | dict[st
 
     In case of corrupt input arguments, None is returned
     """
+    tokens = tokenize(text)
+    freq = calculate_frequencies(tokens)
+    if freq is not None and tokens is not None and isinstance(language, str):
```

Comment:
the same

Reply:
I can't. I have the necessary condition.

Reply:
inverse the condition and return `None` as soon as posible

Reply:
agree with Artyom

Reply:
done, it seems